api/domain/customersDomain.js

```diff
@@ -44,23 +33,20 @@ exports.delete = async (customerId) => {
 exports.getAll = async () => {
     let fetchedRows = await customerRepo.getAll();
 
-    if (fetchedRows === 0) {
-        throw new Error('No customers found.');
+    //IF TABLE IS EMPTY STILL RETURNS INSTEAD OF THROWING AN ERROR
+    if (fetchedRows === [0]) {
+        throw new RecordsEmptyError('No customers found.');
```

Comment:
i think we shouldn't throw an error when getAll returns empty array, because logically thinking it's not an error - getAll does what it is supposed to do correct, it's just no rows available. So maybe let's stick to returning empty array instead of error in this case

Reply:
I wanted to set this error as an informer that the customer table is empty